deploy/standalone/flux-deployment.yaml

```diff
@@ -4,11 +4,25 @@ metadata:
   name: flux
 spec:
   replicas: 1
+
+  # Having maxSurge 0 and maxUnavailable 1 means the deployment will update one
```

Comment:
If we have more than one replica, this will ensure we deploy 1-at-a-time so they stay available while deploying.